lips/lip-6.md

```diff
@@ -0,0 +1,79 @@
+---
+lip: 6
+title: Libra Validator Configuration Management
+author: Valeria Nikolaenko, David Wolinsky, Sam Blackshear, David Dill
+status: Draft
+type: Informational
+created: 07/30/2020
+---
+# Summary
+This LIP describes the conceptual model and implementation of validator configuration management on the Libra blockchain.
```

Comment:
What is the scope?  When I started reading, I thought it was a specification of the Move Framework:  Suppose someone wanted to implement a validator in C++, what would they have to implement?  But it also includes off-chain concepts and constraints.  Is that to explain the programmatic concepts, or are we also specifying the off-chain rules (e.g., there is exactly one validator node per validator operator?).  (This is a discussion point, not a suggestion for rewriting at this point.)

Reply:
The intention I think was to cover the Validator Set management, where the Validator Set is an on-chain config (one of several we set with LibraConfig::set(...)). Including both on-chain guarantees (with some subset that we could formally verify) and off-chain obligations of each of the parties (the Association should make sure there is enough diversity in the Validator Set, that validator operator should run one node per owner, etc.)

If someone wants to implement the validator in C++ they can refer to Libra Specs (https://github.com/novifinancial/libra-specs).